pkgs/tools/X11/smart-wallpaper/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
```

Comment:
```suggestion
{ stdenvNoCC
```

Reply:
Thank you for the review, I have applied your suggestions